scripts/test_docker_image.sh

```
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# test_docker_image.sh - Validates a built Docker image for the Wanderer Notifier application
+# This script performs basic validation to ensure the critical components are working
+
+set -e
+
+# Default values
+IMAGE_NAME="guarzo/wanderer-notifier"
+TAG="latest"
+TIMEOUT=30
```

Shellcheck (0.10.0)

[warning] TIMEOUT appears unused. Verify use (or export if used externally).

(SC2034)